.github/workflows/test.yml

```diff
@@ -161,3 +161,21 @@ jobs:
         command: ssh default -- "CONTAINERD_SNAPSHOTTER=fuse-overlayfs /vagrant/nerdctl.test -test.v -test.kill-daemon"
     - name: "Uninstall rootless containerd"
       run: ssh default -- containerd-rootless-setuptool.sh uninstall
+
+  test-unit-freebsd-amd64:
```

Comment:
@AkihiroSuda Instead of using Cirrus, we can run FreeBSD jobs in GH actions this way. What do you think?